picodom.d.ts

```diff
@@ -0,0 +1,26 @@
+export interface VNode<D> {
+  tag: string
+  data?: D
+  children: VElement<{}>[]
```

Comment:
Yeah it's an array of `VNode | string` right? How else do you represent text nodes?